### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by Ablu**

### CODE REVIEW

File: translations/tiled_de.ts [hunk 1]
```diff
@@ -3725,52 +3726,52 @@ Bitte Format angeben.</translation>
...
         <location line="+71"/>
         <source>Choose the Stamps Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Stempel-Ordner auswählen</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Tile Stamps</source>
-        <translation type="unfinished"></translation>
+        <translation>Kachel Stempel</translation>
```
Comment (Ablu) [inline translations/tiled_de.ts]: This sounds a bit weird to me though... In the context of the window only `Stamps` is used. But here it says `Tile Stamps`

Comment (bjorn) [inline translations/tiled_de.ts]: Feel free to translate it with just "Stempeln".

Comment (bjorn) [inline translations/tiled_de.ts]: Erm... "Stempel".

Comment (Ablu) [inline translations/tiled_de.ts]: Ok done.

**Comment by Ablu** [comment]
@wayfu do you have time for a review maybe?

**Comment by wayfu** [comment]
At a quick glance, just a couple of things.. I'd comment single lines in here if that's okay with you? https://github.com/Ablu/tiled/commit/d2927dbe9377d8cc09364f7f2e842e566088c1cb

**Comment by Ablu** [comment]
Best comment in the diff view of the pullrequest. Then the comments are not gone once I do a forced push and it is easier to track if i caught all issues. But do whatever is the easiest for you.

**Comment by wayfu** [comment]
Since I didn't know who would get how many emails about comments in commit diff, I ended up making a new file and uploading it here: https://dl.dropboxusercontent.com/u/14053711/tiled_de.ts

I hope that agrees with your workflow!

**Comment by Ablu** [comment]
I pushed your changes to this branch/pullrequest, but I have no time to 
look at it in detail today.

**Comment by Ablu** [comment]
Ok your changes look good. I squashed the commits. Should be good to merge now.

Thanks for the review.

**Comment by bjorn** [comment]
Thanks guys!

**Pull request merged**